src/components/pages/cross_word/CrossWordGame/crossword-game.module.css

```
@@ -176,6 +176,30 @@
   }
 }
 
+/* ── Reveal power-up flash — amber ring as each letter lands ── */
+.cellRevealed {
+  animation: revealedPulse 0.45s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+}
+
+@keyframes revealedPulse {
```

Stylelint (17.14.0)

[error] Expected keyframe name "revealedPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)